packages/components/organisms/f-header/src/components/Header.vue

```diff
@@ -91,6 +97,11 @@ export default {
             default: false
         },
 
+        isCondensedOnMid: {
```

Comment:
Does the name of this need to be associated to the breakpoint? Might be worth just calling it `isCondensed` that way if we decide to use it on another breakpoint it is a little more flexible?

Reply:
Yea, makes sense - I'll update that now 😄 I have actually changed a tiny bit of the wide breakpoint, so that will suit better

Reply:
I think adding the point about it is currently used on mid in the readme will be sufficient here :)

Reply:
I added "_Where appropriate_" in the README & `header.stories.js` and kept "_on mid_" in the `changelog.md` 😄